lib/bundler/fetcher/compact_index.rb

```diff
@@ -122,14 +122,24 @@ def call(path, headers)
       end
 
       def md5_available?
+        return true unless fips_enabled? && Process.respond_to?(:fork)
+        pid = fork do
```

Comment:
I don't think forking on every Bundler process is going to be acceptable performance-wise  (in addition to all the platforms that aren't fork-safe) -- is there no way to determine in-process if the MD5 digest is available?